docker-compose.yml

```diff
@@ -0,0 +1,95 @@
+version: '3.7'
+services:
+  http:
+    image: nginx:latest
+    depends_on:
+      - web
+      - api
+    ports:
+      - "80:80"
+    volumes:
+      - "docs:/home/documents"
+      - "./docker/nginx/conf.d/general-configuration.conf:/etc/nginx/conf.d/general-configuration.conf"
+      - "./docker/nginx/snippets/:/etc/nginx/snippets/"
+      - "./docker/nginx/sites-available/readthedocs-main.conf:/etc/nginx/conf.d/default.conf"
+
+  web:
+    hostname: web
+    build: .
+    image: &img web
```

Comment:
I would call the image italia/docs.italia.it:latest (so you're more comfortable if you need to push it to dockerhub)

Reply:
done